### ISSUE TITLE
x509: certificate signed by unknown authority error

### ISSUE THREAD

**Issue by kybin** [raviqqe/muffet#13]
I wanted check my site (https://lazyd.org) with muffet. The site has "COMODO CA Limited" certificate. 

But it exits with `x509: certificate signed by unknown authority` error.

If you know what causes it, please let me know.

Also maybe add -insecure flag would great. Like `git -c http.sslVerify=false`

**Comment by fd0** [comment]
The TLS certificate verification does not work for me either, tested with `curl` and `muffet`. This is not a problem in muffet, but in the configuration of your server: It seems the server does not provide a required intermediate TLS certificate, the trust chain is incomplete. Look at the "certification path" section here: https://www.ssllabs.com/ssltest/analyze.html?d=lazyd.org

**Comment by seigel** [comment]
@fd0 Just wondering what you used to check when using curl.  My site https://atvenu.com is also seeming to fail in muffet but okay in Curl.  Any thoughts?

**Comment by kybin** [comment]
Thank you @fd0 ! I was able to fix it.

Here is the way for reference.

When I got certificate files from COMODO, there was `.ca-bundle` file along with `.crt` and `.key`.
I copied the contents of `.ca-bundle` file and appended it to `.crt`.
After I restart the server, I could both curl and muffet to it.

@seigel I think you've fall into similar problem with me. Does the process help you?

**Comment by fd0** [comment]
@seigel 

```
$ curl https://atvenu.com/

curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.haxx.se/docs/sslcerts.html
curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

You can use ssllabs.com to check for yourself: https://www.ssllabs.com/ssltest/analyze.html?d=atvenu.com&latest

**Comment by gunzip** [comment]
sorry to re-open this but that would be useful some kind of function to skip certificate validation (alike --no-check-certificate in wget)

**Comment by raviqqe** [comment]
Closed by 637733a. Thank you all for your informational discussion!
